plugin.video.filmarkivet/resources/language/resource.language.sv_se/strings.po

```diff
@@ -0,0 +1,54 @@
+# Kodi Media Center language file
```

Comment:
Sorry for so many requests but, for the addon to be in the gotham branch you need to keep the old language layout while moving to gettext:

plugin.video.filmarkivet/resources/language/Swedish/strings.po